src/index.js

```diff
@@ -16,11 +17,33 @@
 import makeHardener from '@agoric/make-hardener';
 import buildTable from './buildTable';
 
-// this use of 'global' is why Harden is a "resource module", whereas
-// MakeHardener is "pure".
-const initialRoots = buildTable((1, eval)('this')); // eslint-disable-line no-eval
-// console.log('initialRoots are', initialRoots);
+// Hunt down our globals.
+const g = typeof globalThis === 'undefined' ? global : globalThis;
 
-const harden = makeHardener(initialRoots);
+// Try to use SES's own harden if available.
+let harden = g.SES && g.SES.harden;
```

Comment:
There is not `SES` global anymore.

Reply:
Is it just `g.harden`?

Reply:
I changed this to be compatible both with legacy `SES.harden` and new `harden`.  It doesn't install anything in the globals anymore.